SU-SWS/stanford_profile_helper
# D8CORE-2951 D8CORE-2952 Style adjustments for people cards

# READY FOR REVIEW

# Summary
- Adjusted styles for person cards with a single per row
- limit person image to 300px.

# Need Review By (Date)
- asap

# Urgency
- high.

# Steps to Test
1. checkout this branch
1. clear browser cache
1. create page with a person content reference as the only 1 per row.
1. verify the card is horizontal.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
